hivemq4/dns-image/Dockerfile

```diff
@@ -3,6 +3,8 @@ ARG BASEIMAGE=hivemq/hivemq4:latest
 
 FROM ${BASEIMAGE}
 
+USER 0
```

Comment:
curious, for what operation are we switching to the root user?

Reply:
for example `chmod` commands